packages/ckeditor5-dev-docs/lib/validators/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const seeValidator = require( './see-validator' );
+const linkValidator = require( './link-validator' );
+const firesValidator = require( './fires-validator' );
+const overloadsValidator = require( './overloads-validator' );
+
+/**
+ * Validates the CKEditor 5 documentation.
+ *
+ * @param {Object} project Generated output from TypeDoc to validate.
+ * @param {Object} typeDoc A TypeDoc application instance.
+ * @returns {Boolean}
+ */
+module.exports = {
+	validate( project, typeDoc ) {
+		const validators = [
+			seeValidator,
+			linkValidator,
+			firesValidator,
+			overloadsValidator
+		];
+
+		let result = true;
+
+		typeDoc.logger.info( 'Starting validation...' );
+
+		for ( const validator of validators ) {
+			const errors = new Set();
+
+			validator( project, error => {
+				result = false;
+
+				errors.add( error );
+			} );
+
+			errors.forEach( error => typeDoc.logger.warn( error ) );
```

Comment:
Is it possible to add a node from the source code where an error occurs? See: https://github.com/ckeditor/ckeditor5-dev/blob/7472b759bc6ccc380dc6016a63ec4a6da737a0ed/packages/typedoc-plugins/lib/tag-event/index.js#L75.

Reply:
I already tried to do that, but it turns out that not all reflections have a symbol: `project.getSymbolFromReflection( reflection )` returns `undefined`. No idea how to get the internal `node` for such reflections.

Reply:
I tried again and this time it worked. Now, all errors found by the validators will be reported along with the node from the source code that has them.

![obraz](https://user-images.githubusercontent.com/56868128/212254251-0c8cc556-27ac-4cc9-9d81-371cf18faf58.png)